xls/examples/hsv2rgb.x

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Fast HSV to RGB conversion.
+//
+// Range:
+//   h: [0, 0x5ff], s: [0, 0xff], v: [0, 0xff]
+//   r: [0,  0xff], g: [0, 0xff], b: [0, 0xff]
+//
+// Ported from https://www.vagrearg.org/content/hsvrgb (MIT license).
+
+pub fn slope(h_fraction: u16, s: u8, v:u8) -> u8{
+  let u':u32 = (v as u32) * ((u32:255 << 8) - ((s as u16) * h_fraction) as u32);
+  let u'' = u' + (u' >> 8);
+  let u''' = u'' + (v as u32);
+  (u''' >> 16)[0+:u8]
+}
+
+pub fn hsv2rgb(h: u16, s: u8, v: u8) -> (u8, u8, u8) {
+  let sextant_:u8 = (h >> 8)[0+:u8];
```

Comment:
Naming convention is that _ is a name to bind to but not read (https://google.github.io/xls/dslx_reference/#identifiers)